docs/diagrams/AddClassDiagram.puml

```diff
@@ -4,16 +4,17 @@ skinparam classAttributeIconSize 0
 hide circle
 
 abstract class "{abstract}\nCommand" {
-    + {abstract} execute(model : Model) : CommandResult
+    + {abstract} execute(Model) : CommandResult
 }
 
 interface Model <<interface>> {
-    + addPerson(Person person) : void
-    + hasPerson(Person person) : boolean
+    + addPerson(Person) : void
```

Comment:
So are we standardizing putting void for void returns? Because i think some of us didnt put anything at all

Reply:
I've updated to remove void